src/v3/src/components/Spinner/Spinner.tsx

```diff
@@ -11,32 +11,27 @@
  */
 
 import { Box, CircularProgress } from '@mui/material';
+import { loc } from 'okta';
 import { FunctionComponent, h } from 'preact';
 
-import { useTranslation } from '../../lib/okta-i18n';
-
 const Spinner: FunctionComponent<{
   label?: string;
   valueText?: string;
 }> = ({
   label,
   valueText,
-}) => {
-  const { t } = useTranslation();
-
-  return (
-    <Box
-      display="flex"
-      flexDirection="column"
-      justifyContent="center"
-      alignItems="center"
-    >
-      <CircularProgress
-        aria-label={t(label || 'loading.label')}
-        aria-valuetext={t(valueText || 'loading.label')}
-      />
-    </Box>
-  );
-};
+}) => (
+  <Box
+    display="flex"
+    flexDirection="column"
+    justifyContent="center"
+    alignItems="center"
+  >
+    <CircularProgress
+      aria-label={label || 'Loading...'}
+      aria-valuetext={valueText || 'Loading...'}
```

Comment:
We still need to translate `Loading...`, right?

Reply:
Going to discuss this with Lester now, but basically I was told that we cant add new translation keys (or that it is extremely difficult), there is no `Loading` translation key for the current widget. I added this so we can avoid the L10_ERROR from showing up but circling back after our discussion, there are a few other places like this

Reply:
okay - not a blocker

Reply:
After talking to Lester, going to file a Jira and add here, we can request new keys to be created but it is a process. Going to document all the new keys that are required along with context, and will need to work with Alex, Deepa and translator for this to be created.